### PR TITLE
Correction of Months' Names In CKB language

### PR DESCRIPTION
**Pull request by kawan97**
Correction of Months' Names In CKB language

Issue: This pull request addresses an issue where the months' names are incorrect in the Carbon package for the CKB  , I've taken the opportunity to update months name for accuracy and clarity.

**Changes Made:**

1. Corrected the months' names to match the correct naming convention.

 **Reasoning:** Accurate month names are crucial for applications relying on time-related functionalities. Furthermore, updating months name enhances the user experience and ensures clarity in the Carbon package.

Proposed Solution: I've made the necessary changes directly in the ckb source code files . Each change is accompanied by a clear and descriptive commit message detailing the nature of the modification.

**Additional Notes:** If there are any concerns or suggestions regarding these changes, please feel free to discuss them in the comments section of this pull request. I'm open to any feedback or further improvements.

### CODE REVIEW

File: src/Carbon/Lang/ckb.php [hunk 1]
```diff
@@ -11,23 +11,25 @@
 
 /*
  * Authors:
- * - Swara Mohammed
+ * - Swara Mohammed and Kawan Pshtiwan
```
Comment (kylekatarnls) [inline src/Carbon/Lang/ckb.php:14]:
```suggestion
 * - Swara Mohammed
 * - Kawan Pshtiwan
```

**Review by kawan97 — COMMENTED** [review]
Change Months' Names And Add Another Author

**Comment by kylekatarnls** [comment]
Thank you, I'll update the tests and merge it.

**Comment by kawan97** [comment]
ok, i'm waiting.

**Pull request merged**